lib/customer-deposit-wallet/http/Cardano/Wallet/Deposit/HTTP/Endpoints.hs

```diff
@@ -0,0 +1,43 @@
+-- |
+-- Copyright: © 2024 Cardano Foundation
+-- License: Apache-2.0
+--
+-- HTTP endpoints.
+--
+module Cardano.Wallet.Deposit.HTTP.Endpoints
+    ( listCustomers
+    , createAddress
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Deposit.HTTP.Types.JSON
+    ( Address
+    , ApiT (..)
+    , Customer
+    , CustomerList
+    )
+import Control.Monad.IO.Class
+    ( liftIO
+    )
+import Servant.Server
+    ( Handler
+    )
+
+import qualified Cardano.Wallet.Deposit.IO as Wallet
+
+{-----------------------------------------------------------------------------
+    HTTP Wallet endpoints
```

Comment:
what about adding explicitly "principle" idea . So something like,
"
The exposed types go through an ApiT type that allows generating of JSON/CBOR instances for these wrapped types. It could be done by "by hand" or by reusing fine types instances 
"

Reply:
Good idea. 👍  I have added a `NOTE [ApiT]`.